packages/component-library/src/tokens.ts

```
@@ -0,0 +1,35 @@
+
+// Provide the same breakpoints in a form usable by CSS media queries
+// {
+//   breakpoint_small: '512px',
+//   breakpoint_medium: '740px',
+//   breakpoint_wide: '1100px',
+// }
+export const tokens: BreakpointsPx = Object.entries(
+  breakpoints,
+).reduce<BreakpointsPx>(
+  (acc, [key, val]) => ({
+    ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)